src/lexers/javascript-lexer.js

```diff
@@ -36,6 +36,15 @@ export default class JavascriptLexer extends BaseLexer {
     }
   }
 
+  setNamespaces(keys) {
+    return this.defaultNamespace
```

Comment:
I'm not a big fan of ternary operators and the code base temps to avoid them. Would you mind use a boring `if() {} else {}` ?